src/components/WindowTopBar.js

```diff
@@ -7,7 +7,6 @@ import CloseIcon from '@material-ui/icons/CloseSharp';
 import Toolbar from '@material-ui/core/Toolbar';
 import AppBar from '@material-ui/core/AppBar';
 import classNames from 'classnames';
-import WindowIcon from '../containers/WindowIcon';
```

Comment:
Can we throw away `containers/WindowIcon` now too? I don't know if it's also a good time to refactor/rename `component/WindowIcon` for its only remaining consumer too...

Reply:
Looking at this a bit. I'm wondering if alternatively we should just import and use `containers/WindowIcon` in `ManifestListItem` rather than just passing through logo. Is there a reason not to just use the container there?

Reply:
Sure 🤷‍♂️

Reply:
I've pushed up a change that uses `containers/WindowIcon` rather than relying on it being passed through.